packages/excalidraw/element/duplicate.ts

```diff
@@ -196,7 +196,7 @@ export const duplicateElements = (
     index: number,
     elements: ExcalidrawElement | null | ExcalidrawElement[],
   ) => {
-    invariant(index !== -1, "targetIndex === -1 ");
+    invariant(index >= 0, "targetIndex must be >= 0");
```

Comment:
@dwelle I'm not sure having this invariant make any sense anymore. The code below handles incorrect indices gracefully.